projects/packages/jetpack-mu-wpcom/src/features/wpcom-plugins/wpcom-plugins.php

```diff
@@ -24,6 +24,34 @@ function wpcom_plugins_show_banner() {
 
 	$asset_file = include Jetpack_Mu_Wpcom::BASE_DIR . 'build/wpcom-plugins-banner/wpcom-plugins-banner.asset.php';
 
+	/**
+	 * Check to see if a string has been translated. This is for the purposes of changing the banner
+	 * copies below, this function and checks can be removed after new copies are translated.
+	 *
+	 * @param string $string The string to check.
+	 * @return bool True if the string has been translated, false otherwise.
+	 */
+	function should_use_new_translation( $string ) { // phpcs:ignore MediaWiki.Usage.NestedFunctions.NestedFunction
+		if ( function_exists( 'wpcom_launchpad_has_translation' ) ) {
```

Comment:
In the future, [core wordpress will have a has_translation function we can use for things like this](https://core.trac.wordpress.org/changeset/59029/). But that is not included until 6.7.0 which is not yet live on atomic sites. So we will piggy back off a local implementation added in the launchpad feature of jetpack-mu-wpcom for now to check if new translations are in.

Reply:
This is clever! I wondered if we should move it out of launchpad for general use, but I guess it'll be replaced by the core function soon anyway.

Reply:
This is clever! I wondered if we should move it out of launchpad for general use, but I guess it'll be replaced by the core function soon anyway.